css/cssom/getComputedStyle-pseudo.html

```diff
@@ -164,19 +167,24 @@
     getComputedStyle(div, ":totallynotapseudo").length == 0,
     "Should return an empty style for unknown pseudo-elements starting with colon");
 }, "Unknown pseudo-elements");
-test(function() {
-  let li = document.querySelector('li');
-  assert_true(
-    getComputedStyle(li, ':marker').length == 0,
-    "Should return an empty style for :marker");
-  assert_true(
-    getComputedStyle(li, 'marker').length != 0,
-    "Should return the element style for marker");
-  assert_equals(
-    getComputedStyle(li, 'marker').color, "rgb(255, 0, 0)",
-    "Should return the element style for marker (ex: color is rgb(255, 0, 0), not rgb(0, 128, 0)");
-  assert_equals(
-  getComputedStyle(li, '::marker').color, "rgb(0, 128, 0)",
-  "Should return the element style for marker (ex: color is rgb(0, 128, 0)");
-}, "Unknown pseudo-element with a known string (ex: marker)");
+
+// If you add a pseudo-element identifier here, don't forget to add the corresponding style rule in
+// <style> above.
+["marker", "view-transition"].forEach(pseudoIdentifier => {
```

Comment:
```suggestion
[
  "backdrop",
  "file-selector-button",
  "first-letter",
  "first-line",
  "grammar-error",
  "highlight(name)",
  "marker",
  "placeholder",
  "spelling-error",
  "thumb",
  "track",
  "view-transition",
  "view-transition-image-pair(name)",
  "view-transition-group(name)",
  "view-transition-old(name)",
  "view-transition-new(name)",
].forEach(pseudoIdentifier => {
```

I would add many of these common ones.

Reply:
first-letter/first-line support the single colon syntax so I could not add those here. thumb/track also felt a bit too early. Added the others though.

Reply:
> first-letter/first-line support the single colon syntax so I could not add those here.

ah right that makes sense, those were added in CSS 2

Reply:
> thumb/track also felt a bit too early.

There's an `assert_implements_optional` so I'm not too worried, but up to you